packages/discovery-provider/plugins/notifications/src/processNotifications/mappers/base.ts

```diff
@@ -110,8 +112,17 @@ export abstract class BaseNotification<Type> {
       name: string
       handle: string
       is_deactivated: boolean
+      profile_picture?: string | null
+      profile_picture_sizes?: string | null
     }> = await this.dnDB
-      .select('user_id', 'name', 'handle', 'is_deactivated')
+      .select(
+        'user_id',
+        'name',
+        'handle',
+        'is_deactivated',
+        'profile_picture',
```

Comment:
Do we have existing notifications which pull profile pictures? Just wondering what happens if the user in question doesn't have one set.